sdk/textanalytics/Azure.AI.TextAnalytics/src/SummarySentence.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// A sentence extracted from an input document by an extractive text summarization
+    /// operation. The service attributes a rank score to it for measuring how relevant
+    /// the sentence is to the input document.
+    /// </summary>
+    public class SummarySentence
+    {
+        internal SummarySentence(ExtractedSummarySentence sentence)
+        {
+            Text = sentence.Text;
+            RankScore = sentence.RankScore;
+            Offset = sentence.Offset;
+            Length = sentence.Length;
+        }
+
+        /// <summary>
+        /// Gets the sentence text as it appears in the input document.
+        /// </summary>
+        public string Text { get; }
+
+        /// <summary>
+        /// Gets the rank score of the sentence, measuring how relevant the sentence is
+        /// to the input document. The value is set between [0.0, 1.0], with higher
+        /// values indicating a higher relevance.
```

Comment:
Do we know for sure that the rank score is between 0.0-1.0? I remember seeing examples showing negative numbers, but maybe that has since been fixed. @mssfang do you know?

Reply:
If I'm not mistaken, the range was mentioned in one of our design meetings, but it's not documented anywhere. We may need to double check with the service. Created https://github.com/Azure/azure-sdk-for-net/issues/22871 for tracking.